usaspending_api/etl/data_diff_tool.py

```diff
@@ -0,0 +1,42 @@
+from gresearch.spark.diff import *
```

Comment:
I don't think this is needed @bkschmidt -- this code that was created to diff two dataframes using a full outer join is completely standalone, yes?

Reply:
Correct, I included the other function that uses gresearch here but given the later discussion, both that and this line should be removed.